context/app/static/js/components/savedLists/SavedListPanel/SavedListPanel.jsx

```diff
@@ -4,7 +4,8 @@ import { Panel } from 'js/shared-styles/panels';
 function getEntityCounts(listSavedEntities) {
   const counts = { Donor: 0, Sample: 0, Dataset: 0 };
   Object.values(listSavedEntities).forEach((entity) => {
-    counts[entity.entity_type] += 1;
+    const { entity_type } = entity;
+    counts[entity_type] ? counts[entity_type] += 1 : counts[entity_type] = 1;
```

Comment:
@john-conroy : This has eslint errors: assignment inside expression can be confusing. I'll use essentially the same logic, but with a real `if`.